app/src/main/java/org/connectbot/data/ConnectBotDatabase.kt

```diff
@@ -66,12 +71,15 @@ import org.connectbot.data.entity.Pubkey
         PortForward::class,
         KnownHost::class,
         ColorScheme::class,
-        ColorPalette::class
+        ColorPalette::class,
+        Profile::class
     ],
-    version = 2,
+    version = 5,
     exportSchema = true,
     autoMigrations = [
-        AutoMigration(from = 1, to = 2)
+        AutoMigration(from = 1, to = 2),
+        AutoMigration(from = 2, to = 3),
+        AutoMigration(from = 4, to = 5)
```

Comment:
This may be annoying to you, but I would expect a PR to only increment the database version once. It will save the millions of ConnectBot installations from doing unnecessary migrations.